relay-server/src/service.rs

```diff
@@ -131,8 +132,10 @@ impl ServiceState {
         let outcome_aggregator =
             OutcomeAggregator::new(&config, outcome_producer.clone()).start_in(&outcome_runtime);
 
+        let (global_config_watch, global_config_receiver) = watch::channel(Arc::default());
```

Comment:
im wondering now, why would we have a subscription message at all, couldn't every service needing a subscirption from any other service receive it in servicestate::start? this would avoid any complexities with messages and async code